src/mon/PGMap.cc

```diff
@@ -925,7 +925,9 @@ void PGMapDigest::dump_object_stat_sum(
     if (verbose) {
       f->dump_int("quota_objects", pool->quota_max_objects);
       f->dump_int("quota_bytes", pool->quota_max_bytes);
+      if (pool->is_tier()) {
```

Comment:
indent?

```suggestion
      if (pool->is_tier()) {
        f->dump_int("dirty", sum.num_objects_dirty);
      }
```

Reply:
updated, thanks!

Reply:
https://github.com/ceph/ceph/pull/40337#issuecomment-868670482, we could add an else to add N/A or 0 here?